docs/eventing/getting-started.md

```diff
@@ -1,163 +1,88 @@
-1. Run the following command to create a namespace called `event-example`:
-
-   ```sh
-   kubectl create namespace event-example
-   ```
+Namespaces are used to group together and organize your Knative resources.
+<!--TODO: Add documentation about namespaces to core docs?-->
 
-   This creates an empty namespace called `event-example`.
+Create a new namespace called `event-example` by entering the following command:
 
-1. Add a Broker named `default` to your namespace with the following command:
-
-   ```sh
-   kubectl create -f - <<EOF
-   apiVersion: eventing.knative.dev/v1
-   kind: Broker
-   metadata:
-     name: default
-     namespace: event-example
-   EOF
-   ```
-
-   This creates the default Broker for the namespace based on config. The Broker
-   is a resource that will allow you to manage your events.
-
-In the next section, you will need to verify that the resources you added in
-this section are running correctly. Then, you can create the rest of the
-eventing resources you need to manage events.
-
-#### Optional Sugar Controller
-
-When installed, the
-[Sugar Controller for Knative Eventing](../install#optional-eventing-extensions)
-will add a controller that reacts to special labels and annotations for your
-resources, you can skip manually creating a Broker in the above step and instead
-label a namespace:
-
-```sh
-kubectl label namespace event-example eventing.knative.dev/injection=enabled
+```
+kubectl create namespace event-example
 ```
 
-This gives the `event-example` namespace the `eventing.knative.dev/injection`
-label, which adds resources that will allow you to manage your events.
-
-### Validating that the `Broker` is running
+## Adding a broker to the namespace
 
-The [`Broker`](./broker/README.md#broker) ensures that every event sent by event
-producers arrives at the correct event consumers. The `Broker` was created when
-you labeled your namespace as ready for eventing, but it is important to verify
-that your `Broker` is working correctly. In this guide, you will use the default
-broker.
+The [broker](./broker/README.md#broker) allows you to route events to different event sinks or consumers.
 
-1. Run the following command to verify that the `Broker` is in a healthy state:
+1. Add a broker named `default` to your namespace by entering the following command:
 
-   ```sh
-   kubectl --namespace event-example get Broker default
-   ```
+    ```
+    kubectl create -f - <<EOF
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+     name: default
+     namespace: event-example
+    EOF
+    ```
 
-   This shows the `Broker` that you created:
+1. Verify that the broker is working correctly, by entering the following command:
 
-   ```sh
-   NAME      READY   REASON   URL                                                        AGE
-   default   True             http://default-broker.event-example.svc.cluster.local      1m
-   ```
+    ```
+    kubectl --namespace event-example get Broker default
```

Comment:
nit number 2.  For folks that are new, might as well give them the shorthand for the namespace:
-n instead of --namespace

Here and throughout.
if we everywhere else always use --namespace, fine :)